src/Routes/misc_routes.php

```diff
@@ -29,6 +29,11 @@
         )
 ));
 
+$misc->add('about-test', new Route(
+    '/about/test',
+    ['_controller' => 'Goteo\Controller\TestController::testAction']
+));
```

Comment:
At the moment, I wasn't able to load controller actions using Symfony Route annotations. Perhaps in another task...